### PR TITLE
Adding a code sample of how the tabs widget can be initialized and used.

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this PR

This PR is an improvement of **Tabs widget**, which provides an example of how the tabs widget can be used and how the widget's options can be passed during initialization. Also the example's result preview is provided.

## Affected URLs

- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_tabs.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_tabs.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_tabs.html

whatsnew
Added a new [code sample](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_tabs.html#code-sample) showing how to initialize and use a tabs widget.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Hi @eduard13. Thank you for your contribution. It looks like the screenshots have retina resolution. As a result, they might not look good on some devices (upscaled)

image

Could you fix the resolution of the images, please? 
Thank  you!

File: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md [hunk 1]
```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
...
+    <div class="item title" data-role="collapsible">
+        <a class="switch" data-toggle="trigger" href="#tab-music">Music</a>
+    </div>
+    <div id="tab-music" class="item content" data-role="content">Music Content</div>
+</div>
+```
+
+### Result
+
+The result are 3 tabs with its content, where the last tab is disabled based on our initialization options, as shown here:
```
Comment (rogyar) [inline guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md]: Please, rephrase the "our initialization" part by removing "our". 
So, in result we have something like "tab is disabled based on the initialization options"

File: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md [hunk 2]
```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
 ### `dimensionsChanged` {#fedg_tabs_dimensionsChanged}
 Called after content is opened or closed.
 
+## Code Sample {#tabs_code_sample}
```
Comment (rogyar) [inline guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md]: Please, remove `{#tabs_code_sample}` part. As it's preferable to use a clean Markdown, the links to the sections can be processed using Markdown.

**Comment by eduard13** [comment]
Hi @rogyar, I've applied the mentioned adjustments and changed the images' resolution. Now it should be looking like: 
Xnip2019-04-15_15-15-34
Thank you.

File: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md [hunk 3]
```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
 ### `dimensionsChanged` {#fedg_tabs_dimensionsChanged}
 Called after content is opened or closed.
 
+## Code Sample
+
+The following example shows how to initialize the tabs widget and pass options during the initialization.
+The classes used are only for getting the same view as the tabs have it on product page.
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md]:
```suggestion
This example uses the same CSS classes as the tabs on the product page.
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md [hunk 4]
```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
...
+    <div class="item title" data-role="collapsible">
+        <a class="switch" data-toggle="trigger" href="#tab-music">Music</a>
+    </div>
+    <div id="tab-music" class="item content" data-role="content">Music Content</div>
+</div>
+```
+
+### Result
+
+The result are 3 tabs with its content, where the last tab is disabled based on initialization options, as shown here:
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md]:
```suggestion
The result is three tabs with content, where the last tab is disabled based on initialization options, as shown here:
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md [hunk 5]
```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
 ### `dimensionsChanged` {#fedg_tabs_dimensionsChanged}
 Called after content is opened or closed.
 
+## Code Sample
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md]:
```suggestion
## Code sample
```

**Comment by eduard13** [comment]
Hi @jeff-matthews, thank you for your review, the changes have been adjusted.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**